administrator/language/de-DE/com_finder.ini

```
@@ -159,6 +159,8 @@ COM_FINDER_MAPS_COUNT_UNPUBLISHED_ITEMS="Gesperrter indexierter Inhalt"
 COM_FINDER_MAPS_MULTILANG="Hinweis: Das Systemplugin „Sprachenfilter“ wurde aktiviert, so dass diese Gruppe nicht verwendet wird."
 COM_FINDER_MAPS_N_ITEMS_DELETED="%d Gruppen gelöscht."
 COM_FINDER_MAPS_N_ITEMS_DELETED_1="Gruppe gelöscht."
+COM_FINDER_MAPS_N_ITEMS_FAILED_PUBLISHING="Fehler beim Veröffentlichen von %d Gruppen, da entweder einer der übergeordneten Einträge versteckt ist oder einer auf derselben Ebene gesperrt ist."
+COM_FINDER_MAPS_N_ITEMS_FAILED_PUBLISHING_1="Fehler beim Veröffentlichen von %d Gruppe, da entweder einer der übergeordneten Einträge versteckt ist oder einer auf derselben Ebene gesperrt ist."
```

```suggestion
COM_FINDER_MAPS_N_ITEMS_FAILED_PUBLISHING_1="Fehler beim Veröffentlichen von %d Gruppe, da entweder einer der übergeordneten Gruppen versteckt ist oder eine auf derselben Ebene gesperrt ist."
```
Einträge vs. Gruppen?